_includes/v21.1/app/create-a-database.md

```diff
@@ -25,35 +25,24 @@
 
 </section>
 
-{% comment %}
 <section class="filter-content" markdown="1" data-scope="cockroachcloud">
 
 1. If you haven't already, [download the CockroachDB binary](install-cockroachdb.html).
-1. Start the [built-in SQL shell](cockroach-sql.html):
+1. Start the [built-in SQL shell](cockroach-sql.html) using the connection string you got from the CockroachCloud Console [earlier](#set-up-your-cluster-connection):
 
     {% include copy-clipboard.html %}
     ~~~ shell
     $ cockroach sql \
-    --url='postgres://<username>:<password>@<global host>:26257?sslmode=verify-full&sslrootcert=<certs_dir>/<ca.crt>'
+    --url='postgres://<username>:<password>@free-tier.gcp-us-central1.cockroachlabs.cloud:26257/<cluster_name>.defaultdb?sslmode=verify-full&sslrootcert=<certs_dir>/cc-ca.crt'
```

Comment:
Is the host name stable/correct here? In the other edits we use `{globalhost}`, and I think we're deploying to AWS now.

Reply:
i'm not sure, i'm seeing this change in 20.2 but `{globalhost}` in 20.1

Reply:
I think we should use `{globalhost}` in place of `free-tier.gcp-us-central1.cockroachlabs.cloud` since (like Ian mentioned), we are allowing users to start Free clusters on AWS now. Can you make sure the placeholder is used in the other examples as well (and updated in v20.2 too)? Thanks!